docs/plugins/convert.rst

```diff
@@ -86,6 +93,13 @@ file. The available options are:
 - **threads**: The number of threads to use for parallel encoding.
   By default, the plugin will detect the number of processors available and use
   them all.
+- **link**: By default, files that do not need to be transcoded will be copied
+  to their destination. This option creates symbolic links instead. Note that
+  options such as ``embed`` that modify the output files after the transcoding
+  step will cause the original files to be modified as well if ``link`` is
+  enabled. For this reason, it is highly recommended not use to ``link`` and
+  ``embed`` at the same time.
+  Default: ``false``.
```

Comment:
There's some identical text between this and the -l option. For maintainability, could you please just leave one copy and refer to one from the other? For example, you might just say "see the description of the `link` option below" or something.